caddy/caddy.go

```diff
@@ -16,7 +16,6 @@ package caddy
 
```

Comment:
We should remove the text in step 2 (lines 7-8, above) about piped config as part of restart.